manifests/baremetal/keepalived.conf.tmpl

```diff
@@ -4,21 +4,18 @@
 # For more information, see installer/data/data/bootstrap/baremetal/README.md
 # in the installer repo.
 
-{{`{{$nonVirtualIP := .NonVirtualIP}}`}}
-
 {{`vrrp_instance {{.Cluster.Name}}_API {
     state BACKUP
     interface {{.VRRPInterface}}
     virtual_router_id {{.Cluster.APIVirtualRouterID }}
     priority 50
     advert_int 1
+    nopreempt
```

Comment:
After testing deployment with this PR, it seems that configuring 'nopreempt' only at bootstrap side doesn't change the behavior, bootstrap moves to one of the master nodes as soon as API is up.
So, I would suggest to delete the nopreempt